include/dso_symbol_lookup.hpp

```diff
@@ -15,8 +15,9 @@ namespace ddprof {
 
 class DsoSymbolLookup {
 public:
-  SymbolIdx_t get_or_insert(ElfAddress_t addr, const Dso &dso,
-                            SymbolTable &symbol_table);
+  SymbolIdx_t get_or_insert(FileAddress_t normalized_addr, const Dso &dso,
```

Comment:
Feedback from Nicolas (not fully relevant to this PR): 
We should do the same as Go for the mappings 
- Single mapping per binary,
Low address is mod low (same for high).
Offset is 0

- We don't need to display these addresses (though they are nice to have within the pprof, for debug purposes)

Reply:
Opening a ticket to store information that allows us to symbolize after capture of stack traces